src/calculateRentalCost.js

```diff
@@ -1,10 +1,22 @@
+const carCosts = 40;
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return carCosts * days - 50;
+  }
+
+  if (days >= 3) {
+    return carCosts * days - 20;
```

Comment:
Similar to the previous comment, avoid using literals for the discount value and the 3-day threshold. Define constants with descriptive names, such as `const SHORT_TERM_RENTAL = 3;` and `const SHORT_TERM_DISCOUNT = 20;`.